src/AutoRest.CSharp/Common/Output/Builders/SerializationBuilder.cs

```diff
@@ -257,26 +257,28 @@ public XmlObjectSerialization BuildXmlObjectSerialization(ObjectSchema objectSch
 
         private IEnumerable<JsonPropertySerialization> GetPropertySerializationsFromBag(PropertyBag propertyBag, SchemaObjectType objectType)
```

Comment:
Not related to this PR, but here we refer to a private type `PropertyBag` within this class and there is another public type [`PropertyBag`](https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/Common/Output/Models/Shared/PropertyBag.cs). It is ambiguous with the same name, consider to rename the private one here.

Reply:
I will consider this.

Reply:
I renamed it to SerializationPropertyBag